auth-headers-manager/handler.lua

```diff
@@ -36,6 +37,23 @@ local function getCookies()
   return cookies
 end
 
+local function getCleanedCookiesValue(cookies)
```

Comment:
Not sure how clear it is from the name that "cleaned" stands for "without folio access token"